spec/features/clearance/user_signs_in_spec.rb

```diff
@@ -0,0 +1,17 @@
+require "rails_helper"
+require "support/features/clearance_helpers"
```

Comment:
Should this be autoloaded by the `spec_helper`?

The snipped would look something like:

```ruby
Dir["./spec/support/**/*.rb"].each { |file| require file }
```